packages/speech/src/v1/speech_client.js

```diff
@@ -106,8 +106,26 @@ function SpeechClient(gaxGrpc, grpcClients, opts) {
   speechStubMethods.forEach(function(methodName) {
     self['_' + methodName] = gax.createApiCall(
       speechStub.then(function(speechStub) {
+
+        // here, speechStub is an instance of this gRPC client
+        // https://github.com/grpc/grpc/blob/master/src/node/src/client.js#L722-L760
+
         return function() {
           var args = Array.prototype.slice.call(arguments, 0);
+
+          // log to console if gRPC client succeeded to connect in 3 seconds or not
+          const CONNECT_TIMEOUT = 3 * 1000;
+          speechStub.waitForReady(Date.now() + CONNECT_TIMEOUT, (err) => {
+            if (err) {
+              console.log(`${methodName}: Failed to connect in ${CONNECT_TIMEOUT}msec`);
+            } else {
+              console.log(`${methodName}: Connected in ${CONNECT_TIMEOUT}msec`);
+            }
+          });
```

Comment:
I just log it to console to show that it works.
Ideally, it should: 
- get CONNECT_TIMEOUT from options (options are 1st argument, args[0])
- throw an error if connection failed within CONNECT_TIMEOUT
- or pass an error to some callback (callback can be passed in options also)